std/mmfile.d

```diff
@@ -344,14 +348,14 @@ class MmFile
                 ? 2*window : cast(size_t) size;
             p = mmap(address, initial_map, prot, flags, fd, 0);
             if (p == MAP_FAILED)
-            {
-                if (fd != -1)
                 {
-                    .close(fd);
-                    fd = -1;
+                    if (fd != -1)
+                        {
+                            .close(fd);
+                            fd = -1;
+                        }
+                    errnoEnforce(false, "Could not map file "~filename);
```

Comment:
Once again, the changeset is cluttered by non changes. The signal to noise ratio is not ideal.